lib/presentation/app_settings/theme_class.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class ThemeClass {
+  static ThemeData lightTheme = ThemeData(primarySwatch: Colors.red);
+
+  static ThemeData darkTheme = ThemeData(
+    colorScheme: ColorScheme.dark(
+        primary: Colors.red[900]!, secondary: Colors.red[900]!, surface: Colors.red[900]!, onSecondary: Colors.white),
```

Comment:
As an idea for the future improvement - have dedicated color palette for the app.